dlp/snippets/src/test/java/dlp/snippets/DeIdentificationTests.java

```diff
@@ -732,6 +732,36 @@ public void testDeIdentifyWithBucketingConfig() throws IOException {
     assertThat(output).contains("Table after de-identification: ");
   }
 
+  @Test
+  public void testDeIdentifyTableWithMultipleCryptoHash() throws IOException {
+
+    Table tableToDeIdentify =
+        Table.newBuilder()
+            .addHeaders(FieldId.newBuilder().setName("userid").build())
+            .addHeaders(FieldId.newBuilder().setName("comments").build())
+            .addRows(
+                Table.Row.newBuilder()
+                    .addValues(Value.newBuilder().setStringValue("user1@example.org").build())
+                    .addValues(
+                        Value.newBuilder()
+                            .setStringValue(
+                                "my email is user1@example.org and phone is 858-555-0222")
+                            .build())
+                    .build())
+            .build();
+
+    // Generate a random string to use it as Transient CryptoKey.
+    String randomString = UUID.randomUUID().toString().replace("-", "");
+    String randomString2 = UUID.randomUUID().toString().replace("-", "");
+
+    DeIdentifyTableWithMultipleCryptoHash.deIdentifyWithCryptHashTransformation(
+        PROJECT_ID, tableToDeIdentify, randomString, randomString2);
```

Comment:
It will be simpler to just pas "TransientKeyName1" and "TransientKeyName2" and leave a comment that DLP API will generate transient keys and use them for hashing.

The use of random made me check how the random UUID is being used as a CryptoKey.